app/src/main/java/com/example/admin/learningandroid/Activity_of_Learning_Activity.java

```diff
@@ -0,0 +1,44 @@
+package com.example.admin.learningandroid;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+public class Activity_of_Learning_Activity extends AppCompatActivity {
+    Button activity_lifecycle, tasks, passing_object;
+
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.layout_of_activity_learning);
+        activity_lifecycle = (Button) findViewById(R.id.activity_lifecycle);
+        tasks = (Button) findViewById(R.id.tasks);
+        passing_object = (Button) findViewById(R.id.passing_object);
+
+
+        activity_lifecycle.setOnClickListener(new View.OnClickListener() {
+            @Override
```

Comment:
Instead of creating multiple listener you can use one 
tasks.setOnClickListener(new View.OnClickListener() {
     @Override
            public void onClick(View view) {
                int id = view.getId();
                // And then use switch case 
           }
        });